lib/cppcheck.cpp

```diff
@@ -622,7 +634,7 @@ unsigned int CppCheck::checkFile(const FileWithDetails& file, const std::string
     if (mSettings.checks.isEnabled(Checks::unusedFunction) && !mUnusedFunctionsCheck)
         mUnusedFunctionsCheck.reset(new CheckUnusedFunctions());
 
-    mExitCode = 0;
```

Comment:
was there reason to remove the mExitCode. I'm not sure why it should be needed by anyway..